Kickstarter-iOS/Views/SettingsNotificationCell.swift

```diff
@@ -0,0 +1,158 @@
+import Foundation
+import Prelude
+import Library
+import KsApi
+
+protocol SettingsNotificationCellDelegate: class {
+  func didFailToSaveChange(errorMessage: String)
+  func didUpdateUser(user: User)
+}
+
+final class SettingsNotificationCell: UITableViewCell, NibLoading, ValueCell {
+  @IBOutlet fileprivate weak var titleLabel: UILabel!
```

Comment:
Could you alphabetise here? (I know ... 😬 )

Reply:
😭